ibis_heavyai/operations.py

```diff
@@ -1082,7 +1094,6 @@ def _udf(traslator, expr):
 # AGGREGATION/REDUCTION
 _agg_ops = {
     ops.HLLCardinality: approx_count_distinct,
-    ops.DistinctColumn: unary_prefix_op('distinct'),
```

Comment:
Is `distinct` are removed from ibis 3?

Reply:
Not exactly, this is in the release notes: `api: Replace t["a"].distinct() with t[["a"]].distinct().` I am unsure how this translate in this package.